test/p2p/client.sh

```diff
@@ -11,7 +11,7 @@ NAME=test_container_$ID
 echo "starting test client container with CMD=$CMD"
 # run the test container on the local network
 docker run -t --rm \
-	-v "$GOPATH/src/github.com/tendermint/tendermint/test/p2p/:/go/src/github.com/tendermint/tendermint/test/p2p" \
+	-v "$GOPATH/src/github.com/tendermint/tendermint/test/p2p/:/go/src/github.com/tendermint/tendermint/test/p2p":Z \
```

Comment:
NOTE: `:Z` is needed for SELinux to allow writes to the mounted volume on host

Reply:
added